lib/c-api/examples/wasmer-capi-examples-runner/src/lib.rs

```diff
@@ -217,12 +217,12 @@ fn test_run() {
                 fixup_symlinks(
                     &[
                         format!("{}/include", config.wasmer_dir),
-                        format!("{}", config.root_dir),
+                        config.root_dir.to_string(),
                     ],
                     &mut log,
                     &config.root_dir,
                 )
-                .expect(&format!("failed to fix symlinks: {log}"));
+                .unwrap_or_else(|_| panic!("failed to fix symlinks: {log}"));
```

Comment:
We should use the previous coe

Reply:
this is a clippy autofix done by clippy --fix --all, it does exactly the same thing, but it doesn't construct a string in the "okay" case. wontfix because it will be reapplied by `clippy --fix` on the next run anyways.